demes/tests/test_graph.rs

```diff
@@ -910,3 +910,35 @@ pulses:
 ";
     let _ = demes::loads(yaml).unwrap();
 }
+
+#[test]
+fn test_size_at() {
+    let yaml = "
+description: Multiple epochs with different size functions
+time_units: generations
+demes:
+- name: deme0
+  epochs:
+  - {end_time: 200, start_size: 100}
```

Comment:
Because 1/2-open intervals can come back to bite us, we should include a test where an ancestral epoch has end size 200, the next epoch had start size 100.  The size at that time should be 200, because the epoch interval is [end_time, start_time).

Reply:
Good point thanks

Reply:
Added a transition test.